api_tester/android/app/build.gradle

```diff
@@ -43,11 +43,12 @@ android {
     }
 
     defaultConfig {
-        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.revenuecat.api_tester"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
-        minSdkVersion flutter.minSdkVersion
+        // TODO: change
+        // minSdkVersion flutter.minSdkVersion
```

Comment:
@tonidero we should be able to change this now, right?

Reply:
Hmm we would need to split the API tests since currently they also have the new package. Not sure if it's worth doing that though?

Reply:
Oh okay makes sense 👍🏻 I'll remove this then.